recipes/eglibc/eglibc_2.18-r24943.oe.sig

```diff
@@ -1 +1,2 @@
+517cfd34b8d250dcdca0f3490fa4e9bf571ee473  svn://www.eglibc.org/branches/eglibc-2_18;rev=r24943
```

Comment:
Hmm...  I wonder why the signatures are different.  Hopefully this is only timestamp that differs.